lib/features/manage_account/presentation/profiles/identities/identities_controller.dart

```diff
@@ -287,8 +306,45 @@ class IdentitiesController extends BaseController {
         AppLocalizations.of(currentContext!).you_are_changed_your_identity_successfully);
     }
 
+    _cleanUpPublicAssets(success.identityId);
+
     _refreshAllIdentities();
   }
 
   bool get isSignatureShow => identitySelected.value != null;
+
+  void _injectCleanUpPublicAssetsInteractorBindings(Session? session, AccountId? accountId) {
+    try {
+      requireCapability(session!, accountId!, [CapabilityIdentifier.jmapPublicAsset]);
+      CleanUpPublicAssetsInteractorBindings().dependencies();
```

Comment:
please verify about mem leak? and make sure we can not get the instance of some components like: CleanUpPublicAssetsInteractor, RemoveIdentityFromPublicAssetsInteractor, ... after identity screen closed

Reply:
Done